charts/logzio-telemetry/values.yaml

```diff
@@ -944,4 +997,61 @@ opencost:
+        custom: container_oom_events_total
+      drop:
+        custom: 
+    # for applications pipeline: applications job
+    applications:
+      keep:
+        custom: avalanche_metric_mmmmm_119_0|avalanche_metric_mmmmm_120_0
+      drop:
+        custom: 
+  
+  # Namespaces names to be filtered
+  namespaces:
+    # for infrastructure pipeline: infrastructure & cadvisor jobs
+    infrastructure:
+      keep:
+        custom:
+      drop:
+        kubeSystem: kube-system # need to also enable the flag: enableMetricsFilter.kubeSystem=true
+        custom:
+    # for applications pipeline: applications job
+    applications:
+      keep:
+        custom:
+      drop:
+        custom:
+  
+  # Services names to filtered
+  services:
```

Comment:
shouldn't this be global filter and not per job?

Reply:
Its not needed for applications job (not using services) , will fix it.